distribution/tools/launchers/src/main/java/org/elasticsearch/tools/launchers/SystemJvmOptions.java

```diff
@@ -59,12 +59,7 @@
             "-Dio.netty.allocator.numDirectArenas=0",
             // log4j 2
             "-Dlog4j.shutdownHookEnabled=false",
-            "-Dlog4j2.disable.jmx=true",
-            /*
-             * Due to internationalization enhancements in JDK 9 Elasticsearch need to set the provider to COMPAT otherwise time/date
-             * parsing will break in an incompatible way for some date patterns and locales.
-             */
-            "-Djava.locale.providers=COMPAT"));
```

Comment:
Why can't we keep this in system jvm options? Shouldn't this be something a user doesn't need to think about?

Reply:
you are right, it is better to stay in `SystemJvmOptions`. We can always check the version running with `JavaVersion.majorVersion(JavaVersion.CURRENT) == 8` to choose the right settings